crates/sui-graphql-rpc/schema/draft_schema.graphql

```diff
@@ -177,6 +177,20 @@ scalar Base64
 # ISO-8601 Date and Time
 scalar DateTime
 
+# Scalar representing the contents of a Move Value, corresponding to
+# the following recursive type:
+#
+# type MoveData =
+#     { Number:  BigInt }
+#   | { Bool:    bool }
+#   | { Address: SuiAddress }
+#   | { UID:     SuiAddress }
+#   | { String:  string }
+#   | { Vector:  [MoveData] }
+#   | { Option:   MoveData? }
+#   | { Struct:  [{ name: string, value: MoveData }] }
+scalar MoveData
```

Comment:
@Jordan-Mysten  / @hayes-mysten , I'm particularly curious to get your take on the decision to represent this as a scalar, rather than an output type.  When I was imagining how people would care to consume the move data, I found it almost impossible to use when it required querying through the nested structure of an output type, vs having a scalar for this.

Reply:
I like this change.  I can't think of a situation where you wouldn't want to select all of the data, and manually writing that query every time is a pain